src/components/DpUploadFiles/DpUpload.vue

```diff
@@ -267,6 +267,15 @@ export default {
       this.currentFileId = ''
       this.$emit('upload-success', newFile)
     })
+
+    // Access the hidden input element, that accepted files array.
+    const uppyInputs = document.querySelectorAll('.uppy-DragDrop-input')
+    if (uppyInputs) {
+      uppyInputs.forEach((uppyInput, index) => {
+        // Add data-cy attribute
+        uppyInput.setAttribute('data-cy', `uppyDragDropInput:${index}`)
+      })
+    }
```

Comment:
I think this should be within the initialize-Method. Otherwise it could fire too early.